api/notd/store/schema.py

```diff
@@ -53,5 +53,7 @@
     sqlalchemy.Column(key='openseaSlug', name='opensea_slug', type_=sqlalchemy.Text, nullable=True),
     sqlalchemy.Column(key='url', name='url', type_=sqlalchemy.Integer, nullable=True),
     sqlalchemy.Column(key='discordUrl', name='discord_url', type_=sqlalchemy.Text, nullable=True),
-    sqlalchemy.Column(key='bannerImageUrl', name='banner_image_url', type_=sqlalchemy.Text, nullable=True)
+    sqlalchemy.Column(key='bannerImageUrl', name='banner_image_url', type_=sqlalchemy.Text, nullable=True),
+    sqlalchemy.Column(key='doesSupportErc721', name='does_support_erc_721', type_=sqlalchemy.Boolean, nullable=True),
+    sqlalchemy.Column(key='doesSupportErc1155', name='does_support_erc_1155', type_=sqlalchemy.Boolean, nullable=True)
```

Comment:
goo to have a trailing comma on lines like this so you dont have to change them when adding another